packages/components-native/src/Typography/Typography.style.ts

```diff
@@ -88,6 +88,11 @@ export const typographyTokens: { [index: string]: TextStyle } = {
     fontStyle: "italic",
   },
 
+  underline: {
+    textDecorationColor: tokens["color-base-grey--300"],
```

Comment:
This needs to be a semantic token - we shouldn't be specifying base values anywhere in an Atlantis component unless there's a very specific case where we don't want it to respond to theming

Reply:
Would `tokens["color-text--secondary"]` be acceptable @chris-at-jobber ?

Reply:
What is the expected experience if a consumer is using a variation of Typography? (ie `critical`); do we expect that this color should respond to the color of the text itself?

Reply:
Another example would be places where we are implementing a "link-looking" element (which some teams have done on their own with custom solutions), the underline uses `color-interactive` to match the text element.

Reply:
@sabidhasan I would try 

```suggestion
    textDecorationColor: tokens["color-border"],
```

as a starting point

Reply:
I'll let @jlelford answer here, as the designs explicitly do _not_ use the same color for the underline as they do for the text.

Reply:
@chris-at-jobber Abid was trying to find a token I was using in Figma that doesn't seem to align correctly in Atlantis. I've since changed the request to be `var(--color-text--secondary)` 

FYI I was originally using `border--interactive` which seems to be much darker in Figma than what is listed in Atlantis `var(--color-base-blue--300)`

Reply:
It took me awhile to reply so I missed this conversation, I intentionally had the dotted line a different colour from the text in the desktop version in an effort to differentiate the action as a tooltip/popover instead of a regular link.

I'd used [Polaris' tooltip](https://polaris.shopify.com/components/overlays/tooltip?example=tooltip-with-underline) underline as inspiration.

Wouldn't `color-border` be difficult to differentiate on top of the already light grey message background here?